nautobot_golden_config/models.py

```diff
@@ -138,7 +138,9 @@ def _verify_get_custom_compliance_data(compliance_details):
 
 def _get_hierconfig_remediation(obj):
     """Returns the remediating config."""
-    hierconfig_os = HIERCONFIG_LIB_MAPPER_REVERSE.get(obj.device.platform.slug)
```

Comment:
```
hierconfig_os = HIERCONFIG_LIB_MAPPER_REVERSE.get(get_platform(obj.device.platform.slug))
```

Reply:
@mzbroch It seems to be working with both `ios` and `cisco_ios` as the platform now.

Reply:
thx